eventsourcing/src/main/scala/com/evolutiongaming/akkaeffect/eventsourcing/Engine.scala

```diff
@@ -183,12 +205,13 @@ object Engine {
     }
 
     for {
-      stateRef    <- Ref[F].of(State(initial, stopped = false)).toResource
-      append      <- Append.of(append).toResource
-      cores       <- Runtime.summon[F].availableCores.toResource
-      parallelism  = (cores max 2) * 10
-      queue       <- queue(parallelism, stateRef, append)
-      engine       = {
+      optimisticRef <- Ref[F].of(State(initial, stopped = false)).toResource
```

Comment:
tbh, I'd leave `stateRef` without renaming, as this is used for tracking `internal state`

Reply:
done